api/pkg/transformer/feast/transformer.go

```diff
@@ -248,12 +248,13 @@ func (t *Transformer) getFeatureFromFeast(ctx context.Context, entities []feast.
 		}
 		batchedEntities := entityNotInCache[startIndex:endIndex]
 
-		go func(project string, entityList []feast.Row, columns []string) {
+		go func(project string, entityList []feast.Row, columns []string, i int) {
```

Comment:
It seems the variable `i` is not used?

Reply:
You're right. It was added for debugging purposes. Removed.